Sources/Workspace/Workspace.swift

```diff
@@ -386,6 +389,7 @@ public class Workspace {
         config: SwiftPMConfig = SwiftPMConfig(),
         fileSystem: FileSystem = localFileSystem,
         repositoryProvider: RepositoryProvider = GitRepositoryProvider(),
+        checksumAlgorithm: HashAlgorithm = SHA256(),
```

Comment:
Can you switch this to SHA256 from CryptoKit if it's available?

Reply:
i.e.:

```
if @available(macOS 10.15, *) {
    return CryptoKit.SHA256(..)
} else {
    return TSCBasic.SHA256(..)
}
```

Reply:
I've updated the PR to use CryptoKit.SHA256 on macOS 10.15.